app/src/main/java/pl/futurecollars/invoicing/utils/FileService.java

```diff
@@ -2,15 +2,27 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.List;
+import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.io.FileUtils;
 
+@Slf4j
 public class FileService {
 
     private final File file;
 
     public FileService(String fileName) {
         file = new File(fileName);
+        try {
+            Files.createFile(Paths.get(fileName));
+        } catch (FileAlreadyExistsException e) {
+            log.info(e.getMessage());
+        } catch (IOException e) {
+            throw new RuntimeException();
```

Comment:
"implementation of RuntimeException" :) create e.g. FileNotCreatedException that extends RuntimeException and throw it here